packages/styles/src/components/forms.scss

```diff
@@ -92,16 +58,17 @@ select.form-control-rg:not([size]):not([multiple]) {
 
   @include utilities.disabled-style {
     ~ .form-text {
-      color: var(--post-gray-60);
+      color: tokens.get('input-color-disabled-fg');
     }
   }
 
   &[readonly]:not(:disabled) {
-    border-color: forms.$input-color;
-    background-color: forms.$input-bg;
+    border-color: tokens.get('input-color-enabled-border');
+    background-color: tokens.get('input-color-enabled-bg');
   }
 
   &[type='file'] {
```

Comment:
The value is not correctly aligned for the file input.

Reply:
I created a separate ticket for that: https://github.com/swisspost/design-system/issues/4001